src/Cake.Common/Tools/OpenCover/OpenCoverSettings.cs

```diff
@@ -44,6 +44,24 @@ public ISet<string> ExcludedFileFilters
         }
 
         /// <summary>
+        /// Gets or sets the register option
+        /// </summary>
+        public string Register
+        {
+            get;
+            set;
+        }
+
+        /// <summary>
+        /// Gets or sets the Return target code offset to be used
+        /// </summary>
+        public int? ReturnTargetCodeOffset
```

Comment:
Should be `public int? ReturnTargetCodeOffset { get; set; }` to be more like other aliases.